docs/setup/set_and_series_patch.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Patch simulations
+nav_order: 1
+parent: Simulation set and series
+---
+
+# Patch simulation set and series
+
+A group of simulation series is called a `set`.
+A setup file can only contain one `set`.
+A simulation `series` is a group of simulations that only differ in the random seed.
```

Comment:
It would be helpful to include an example/explanation for how to set up multiple series.

This relates to a conversation with @cainja where someone mentioned having the equivalent of running a bunch of simulations where we go through a list of parameter values for one parameter, and if there was a way to do that non-manually. I think this feature will be relevant for people who use the model to see how changing the range of a parameter affects the resulting behavior.

Reply:
Added examples for setting up simulations with one or multiple series.

The current non-manual way to do parameter sweeps would be to set up a template setup file and then populate it with parameter values using something like Jinja (see the [run batch simulations flow](https://github.com/allen-cell-animated/cell-abm-pipeline/blob/main/src/cell_abm_pipeline/flows/run_batch_simulations.py) for a pipeline example).

I'm not sure that ARCADE itself should support something like `<population.parameter id="A" value="(1|2|3)" />`. It would be straightforward for a single parameter, but I think parsing would get a lot more complex if multiple parameters are changed. For example, for something like:

```
<population.parameter id="A" value="(1|2|3)" />
<population.parameter id="B" value="(4|5|6)" />
```

would this is be 3 or 9 combinations? We should probably move this to an issue. @cainja?

Reply:
I don't necessarily think this approach itself needs to be a feature - many other libraries do sensitivity analysis setup and file editing well, and if you're doing many simulations parallelizing input file creation/running simulations is more important than having one setup file. I don't think manually writing sensitivity analysis parameters is the solution. 

That being said, my opinion would be that type of input should be 3 combinations of [(A=1, B=4), (A=2, B=5), (A=3, B=5)]. Not all SA use combinatorial approaches, e.g. [morris method](https://en.wikipedia.org/wiki/Morris_method). Another thing to consider is whether or not these would be run for all seeds.

Reply:
See #127